src/Microsoft.Azure.WebJobs.Extensions.Dapr/Utils/EnvironmentUtils.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.WebJobs.Extensions.Dapr.Utils
+{
+    using System;
+    using Microsoft.Azure.WebJobs;
+
+    internal static class EnvironmentUtils
+    {
+        public static bool IsAppService(INameResolver nameResolver)
```

Comment:
I wonder if we should create an issue with WebJobs SDK to expose these functions?

Reply:
Thats a good idea, I will create an issue.